src/Tizen.NUI.Components/Controls/Navigation/Page.cs

```diff
@@ -89,6 +104,38 @@ internal set
             }
         }
 
+        /// <summary>
+        /// Transition properties for the transition of Views in this page during this page is pushed to Navigator.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TransitionBase AppearingTransition
```

Comment:
could you check the https://code.sec.samsung.net/jira/browse/GRE-2025 ?
there seems to be an issue if the `TransitionBase AppearingTransition` used as like : `page.AppearingTransition.AlphaFuction = xxx or page.AppearingTransition.TimePeriod = xxx`
this is why GRE-2025 deprecated setters and use only parameter constructor.
I think it looks late because TransitionBase is already opened as public.
but anyway, I just want to talk about the old hot issue which is similar to this. thank you.

Reply:
HAT!!!!!!!!!!!
Sorry I didn't know the history.
I think it didn't create such a big problem in the ordinary use case of transition fortunately.
But I will remember that......
Thank you.